front/src/applications/operationalStudies/components/ManageTrainSchedule/Itinerary/DisplayItinerary/Destination.tsx

```diff
@@ -21,20 +22,58 @@ import { makeEnumBooleans } from 'utils/constants';
 
 import InputSNCF from 'common/BootstrapSNCF/InputSNCF';
 import { store } from 'Store';
-import { MODES, STDCM_MODES } from 'applications/operationalStudies/consts';
+import {
+  DEFAULT_MODE,
+  DEFAULT_STDCM_MODE,
+  MODES,
+  PointOnMap,
+  STDCM_MODES,
+} from 'applications/operationalStudies/consts';
+import { noop } from 'lodash';
 
 interface DestinationProps {
   zoomToFeaturePoint: (lngLat?: Position, id?: string, source?: string) => void;
+  stdcmMode?: symbol;
+  mode?: string;
+  destination?: PointOnMap | undefined;
+  destinationDate?: string | undefined;
+  destinationTime?: string | undefined;
+  dispatch?: Dispatch;
+}
+
+export function withOSRDData<T>(Component: ComponentType<T>) {
+  return (hocProps: T) => {
```

Comment:
better to have named function for the react dev tool tree

Reply:
It has been renamed